Declare "ch" as int to match getopt return type. Fix apcaccess behavior on
arm.
--- a/src/apcaccess.c
+++ b/src/apcaccess.c
@@ -146,7 +146,7 @@
 #endif
 
    // Process standard options
-   char ch;
+   int ch;
    while ((ch = getopt(argc, argv, "f:h:p:u")) != -1)
    {
       switch (ch)
